src/common/helper/posting.ts

```diff
@@ -42,7 +42,7 @@ export const createPermlink = (title: string, random: boolean = false): string =
 
 export const extractMetaData = (body: string): MetaData => {
     const urlReg = /(\b(https?|ftp):\/\/[A-Z0-9+&@#/%?=~_|!:,.;-]*[-A-Z0-9+&@#/%=~_|])/gim;
-    const userReg = /(^|\s)(@[a-z][-.a-z\d]+[a-z\d])/gim;
+    const userReg = /(^|[ \t]{2,})(@[a-z][-.a-z\d]+[a-z\d])(\/?[-a-z\d]*)/gm;
```

Comment:
This regex seem wrong and only catches few cases, I don’t think we need any change on regex here, because it is only responsible for adding mentions on post json-metadata.